python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -597,18 +565,8 @@ def test_timedelta_series_ops_with_cudf_scalars(data, cpu_scalar, dtype, op):
         [1000000, 200000, 3000000],
         [1000000, 200000, None],
         [],
-        pytest.param(
-            [None],
-            marks=pytest.mark.xfail(
-                reason="https://github.com/pandas-dev/pandas/issues/35644"
-            ),
-        ),
-        pytest.param(
-            [None, None, None, None, None],
-            marks=pytest.mark.xfail(
-                reason="https://github.com/pandas-dev/pandas/issues/35644"
-            ),
-        ),
```

Comment:
version

Reply:
This was not a bug, now our behaviour is in-line with pandas.